jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/LocalBaseImageStepsTest.java

```diff
@@ -84,6 +84,22 @@ public void tearDown() {
     tempDirectoryProvider.close();
   }
 
+  public static int getJavaVersion() {
```

Comment:
Is there a good place to move this common test code somewhere?

Reply:
yeah you could probably put it in `jib-core/src/test/java/com/google/cloud/tools/jib/jre/Versions.java` or something